src/Kros.AspNetCore/Exceptions/ResourceIsForbiddenException.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+namespace Kros.AspNetCore.Exceptions
+{
+    /// <summary>
+    ///  The exception which is thrown when user does not have permission for requested resource.
+    /// </summary>
+    public class ResourceIsForbiddenException : Exception
+    {
+        /// <summary>
+        /// Ctor.
```

Comment:
Inconsistent contructor naming. Here we have only `ctor` but in class above is much longer description. Do we mind?